src/views/common/Animation/FrameAnimation.tsx

```diff
@@ -36,8 +39,21 @@ const setupReplayRate = (realTimeReplayRateMs: number | undefined, animationStat
 }
 
 
-const FrameAnimation = <T, >(props: FrameAnimationProps<T>) => {
-    const { width, height, controlsHeight, state, dispatch, dataSeriesFrameRateHz, children } = props
+const _handleBookmark = (callback?: (state: AnimationState<any>) => void) => {
+    const fn = callback === undefined ? () => {} : callback
+    return fn
+}
+
+
+const _checkBookmark = (callback?: (state: AnimationState<any>) => boolean) => {
+    const fn = callback === undefined ? () => false : callback
+    return fn
+}
```

Comment:
These are just a convenience, since the callbacks might not be defined (as the entire functionality is optional).

Reply:
(Not sure if the anonymous functions here are going to result in needless refreshes/rewrites, we should think about that at some point)